lnrpc/rpc.proto

```diff
@@ -1326,6 +1376,8 @@ message QueryRoutesRequest {
 }
 message QueryRoutesResponse {
     repeated Route routes = 1 [json_name = "routes"];
+
+    string chain =2 [json_name = "chain"];
```

Comment:
Add space after `=`

Reply:
Actually in looking at this again, I think this should also be on the request and not the response. We want to specify which chain we want to query routes for in the request, but on the response I don't think we need this info.

Reply:
IMHO it is better to be included in the response so that the user will have more understanding. 

It is also needed in the response for atomic swap (route involves multiple chains).